tests/behat/src/CmsUiContext.php

```diff
@@ -77,10 +77,49 @@ public function iShouldSeeTheCms()
 
     /**
      * @Then /^I should see a "([^"]*)" notice$/
+     * @deprecated 4.7.0 Use `iShouldSeeAToast` instead
      */
     public function iShouldSeeANotice($notice)
     {
-        $this->getMainContext()->assertElementContains('.notice-wrap', $notice);
+        $this->getMainContext()->assertElementContains('.toast, .notice', $notice);
```

Comment:
This will allow behat test in older version to still work. But ideally for future releases we should switch to using `iShouldSeeAToast`

Reply:
Should this be `.notice-wrap`?

Reply:
:see_no_evil: 
```suggestion
        $this->getMainContext()->assertElementContains('.toast, .notice-wrap', $notice);
```